cli/src/cli.rs

```diff
@@ -76,28 +84,13 @@ pub enum Commands {
     /// Clone a remote Simperby repository to the current directory,
     /// and initialize a new Simperby node after verification.
     ///
-    /// This is same as `git clone && simperby init`.
+    /// This is same as `git clone && cd <directory> && simperby init`.
```

Comment:
What is <directory> here?

Reply:
You need to go into the cloned directory.